site/_posts/2021-01-14-softozor-fusionauth.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: blog-post
+title: Softozor integrated FusionAuth with Hasura and Kubernetes to save development effort
+description: While working on a consulting project building an ecommerce platform, Softozor choose FusionAuth to manage their authentication.
+author: Dan Moore
+image: blogs/...
+category: blog
+tags: topic-community-story
+excerpt_separator: "<!--more-->"
+---
+
+Dr Laurent Michel is a FusionAuth community member and product owner at Softozor. He chatted with us over email about how he and his team are using FusionAuth to meet their auth needs. 
+
+<!--more-->
+
+*This interview has been lightly edited for clarity and length.*
+
+-------
+
+**Dan:** Tell me a bit about your work as a developer, software architect and product owner as well as what softozor does.
+
+**Laurent:**  Softozor is a very little company made up by me, my brother and a few friends. Currently, we work together in our free time. We are mostly physicists and mathematicians. My brother and I work in software development companies. Our friends are teachers. One of the goals of the Softozor company is to produce educational video games. In order to achieve that goal, we need money, which we are trying to earn with software development projects. Currently, we are developing an ecommerce platform helping craftsmen to sell their products and make them visible in the regions they are living in. A craftsman can be a farmer, a caterer, a baker, a butcher, etc.
+
+I am the company's official leader. I organize the developer, architect, and product owner roles. I am the only one in the team to have the role of product owner. The roles of developer and architect are taken by other members of the team. Every architectural decision is discussed in the team so that we get as much insight as possible into the possible problems that we might face during the development. This way we also spread the knowledge of everyone across the whole team.  
+
+Nowadays, I essentially develop in C++, C# (.Net / WPF), javascript (VueJs), and python (mainly to write acceptance tests).  
+
+> The ease of installation and the set of features were the most decisive motivations to use [FusionAuth].
+
+**Dan:** Is the craftsman site live? Can you share the URL? 
+
+**Laurent:**  Yes, the [craftsman site is live](https://www.ameising.ch), in a reduced form. In the near future, it will show our customers all the shops we are managing. Currently, no shop is shown yet but they will soon pop on the map that we see on the homepage. The platform is not a traditional e-shop. It manages multiple e-shops where each e-shop has multiple tenants. What you see today is just kind of our crowd-funding: we sell baskets of vegetables, meat, fruits, bread, etc. Also, the project you see on that URL only targets Switzerland, no other country. It is a local project focusing on Switzerland. 
```

Comment:
After review of the people who we build the "ameising" website for, it turns out we sell baskets as written here but they don't contain meat. We will of course sell meat over this platform, but not now in the crowd-funding baskets. So I would remove "meat, " from the sentence.